snapd-glib/snapd-snap.c

```diff
@@ -902,9 +960,27 @@ snapd_snap_get_version (SnapdSnap *self)
  * Returns: a website URL.
  *
  * Since: 1.50
+ * Deprecated: 1.66
```

Comment:
Change to `Deprecated: 1.66: Use snapd_snap_get_website_urls()` - this helps people migrate